.github/workflows/default.yaml

```diff
@@ -121,3 +121,37 @@ jobs:
         with:
           name: build-logs-${{ runner.os }}
           path: btlayer-*.log
+
+  static-analysis:
+
+    strategy:
+      fail-fast: true
+
+    runs-on: [ ubuntu-22.04 ]
+
+    permissions:
+      security-events: write
+
+    steps:
+
+      - name: Install Squid prerequisite Linux packages
+        if: runner.os == 'Linux'
+        run: |
+          # required for "apt-get build-dep" to work
+          sudo sed --in-place -E 's/# (deb-src.*updates main)/  \1/g' /etc/apt/sources.list
+          sudo apt-get --quiet=2 update
+          sudo apt-get --quiet=2 build-dep squid
+
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Initialize CodeQL
+        uses: github/codeql-action/init@v2
+
+      - name: Build Squid
+        run: |
+          ./bootstrap.sh
```

Comment:
This is a bug in bootstrap.sh. The warning has shown up on every Linux bootstrap.sh execution since a decade ago without causing the cited confusion. It is out of scope for this PR.

Reply:
> The warning has shown up on every Linux bootstrap.sh execution since a decade ago without causing the cited confusion.

The warning has already caused the cited problems -- I spent time adjusting the now-official workflow configuration to install the tools necessary to eliminate that warning when the builds were failing for other reasons. You do not have to fix or work around this bug, of course, but rejecting the very existence of the problem only wastes time.